engines/freescape/gfx_opengl_shaders.cpp

```diff
@@ -96,7 +96,9 @@ void OpenGLShaderRenderer::init() {
 	_bitmapShader->enableVertexAttribute("position", _bitmapVBO, 2, GL_FLOAT, GL_TRUE, 2 * sizeof(float), 0);
 	_bitmapShader->enableVertexAttribute("texcoord", _bitmapVBO, 2, GL_FLOAT, GL_TRUE, 2 * sizeof(float), 0);
 
+#if !USE_FORCED_GLES2
```

Comment:
Please revert this change, the use of GL_LIGHTING here is actually not necessary and I will remove it in another commit.

Reply:
Great, I removed this. Let me know once it also has been removed in master and I'll rebase this branch and rebuild the demo.

Reply:
Done: https://github.com/scummvm/scummvm/commit/47eb539c3d1381211523d4e1a296c3073e2c3700